src/main/resources/scripts/pipeline.sh

```diff
@@ -0,0 +1,49 @@
+pipeline {
+    agent {
+        label 'main-node'
```

Comment:
We should set environment like [this](https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/opensearch/perf-test.jenkinsfile#L11) and read the values from there.